### PR TITLE
remove dublicated methods

### PR DESCRIPTION
**Pull request by tfe2012**

### CODE REVIEW

**Comment by tautrimas** [comment]
@lmikelionis can you give us information, on why `Tests/app/data/dummyData.sql` was created? Is it hand crafted sql data for additional testing?

**Comment by lukasvtechceo** [comment]
@tautrimas: no idea, wasn't me, i haven't work with OXID...

File: Tests/Functional/AbstractTestCase.php [hunk 1]
```diff
@@ -26,164 +26,19 @@
 /**
  * Class AbstractTestCase.
  */
-abstract class AbstractTestCase extends WebTestCase
+abstract class AbstractTestCase extends ParentTestBase
```
Comment (saimaz) [inline Tests/Functional/AbstractTestCase.php:29]: The right name for all duplicated classes name extensions aliases is `Base`.

p.s. there are many more other cases with different alias naming

**Comment by saimaz** [comment]
Please leave schema and data sql insertions in different files. Also for testing do you need the whole tables? Please remove not necessary from `database.sql`.

**Pull request merged**